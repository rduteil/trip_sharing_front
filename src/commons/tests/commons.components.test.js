import React from "react";
import { shallow } from "enzyme";

import Background from "../components/background";

import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("components rendering", () => {
  it("render background", () => {
    let component = shallow(<Background />);
    expect(component).toMatchSnapshot("./snapshots/commons.components.test.js.snap");
  });
});
