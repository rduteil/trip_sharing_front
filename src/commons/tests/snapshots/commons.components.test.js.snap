// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background renders correctly`] = `
  <div className="FirstLayer">
    <div className="SecondLayer" />
  </div>
`;